FAQ.md

```diff
@@ -77,38 +77,63 @@ Post quality is judged by the amount of effort you put into the creation of your
 ### Q: What are user groups and their privileges?
 **A**: Usergroups are what separates the forum's member by ranking or privileges and consists of:
 
-- ![REGISTERED](https://i.imgur.com/39fVOFD.png) - Default user group after registration 
-  >Privileges:
-  >>Can give 10 likes/day
+- ![NOOB](https://i.imgur.com/FOsw2e9.png) - Demotion from REGISTERED usergroup.
+  >Disadvantages:
+  > - No like button is visible to for any post. Need to upgrade to Boomer by having 10 pearl and 3 post. Any kind of posts count, not just thread.
```

Comment:
Grammar- "No like button is visible to for any post" > "Any posts will not have a like button."
"by having 10 pearl and 3 post" > "By having 10 pearls and 3 posts"
"not just thread." > "not just threads"